src/services/permissions-checker.js

```diff
@@ -16,6 +37,9 @@ function PermissionsChecker(environmentSecret, renderingId, collectionName, perm
       return false;
     }
 
+    if (permissionName === 'smart action') {
```

Comment:
Why "smart action" and not "actions" to be consistent with the collection permission format?

Reply:
it will also be easier for developers to write "action" than "smart action" (less typo prone)

Reply:
Ok I made the changes 👍